### ISSUE TITLE
getting crashes in ARProcessor::addAnchor 

### ISSUE THREAD

**Issue by ofZach** [sortofsleepy/arkit-tests#1]
I was trying to add anchors but having issues with crashes in ARProcessor::addAnchor.

also, if it's helpful, I made a small test program using your code (and whatever I could get to work) to see something move -- 

https://twitter.com/zachlieberman/status/895404101936308224
https://dl.dropboxusercontent.com/u/92337283/misc/arkit-tests_doingSomething.zip

it's pretty messy, but there's some functions you might find useful like getting the modelview and projection matrix out and converting to OF style matrices, etc.  I'll try to to clean it up shortly...

**Comment by sortofsleepy** [comment]
Hey @ofZach ! - 

Again (I responded on the oF forum), thanks for sharing! I'll definitely check it out ASAP and see if theres anything helpful that could be quickly rolled in (assuming you don't get to a PR first) 

As to your `ARProcessor::addAnchor` issue - I ran into a problem as well and assuming you were having the same issue, I just pushed a fix that I think(?) should fix things. Looks like you still need to pull the current frame from the session when attempting to add a new anchor and I mistakenly thought `update()` would run fast enough to grab the new frame

That's actually an area I haven't quite begun to explore in depth just yet so the code you see there is pretty much the same example code XCode 9 generates in it's new project wizard. You're jumping ahead of me there in that area haha.
